examples/asgilook/asgilook/store.py

```diff
@@ -0,0 +1,98 @@
+import asyncio
+import datetime
+import io
+import os.path
+
+import aiofiles
+import falcon
+import PIL.Image
+
+
+class Image:
+
+    def __init__(self, config, image_id, size):
+        self.config = config
+        self.image_id = image_id
+        self.size = size
+        self.modified = datetime.datetime.utcnow()
+
+    @property
+    def path(self):
+        return os.path.join(self.config.storage_path, self.image_id)
```

Comment:
Not 100% sure it is worth it tbh.
Because one of the main uses of this property is to point the `Location` header to it. Using `Path` would probably need to involve a type cast.